.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Usually, one would add `.vscode` to the .gitignore and not include in source control, but feel free since this is your repository.

Reply:
Ok, I'll change that. Thanks.